jest.config-e2e.js

```diff
@@ -3,6 +3,10 @@ module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
   testTimeout: 30000,
+  transform: {
+    '^.+\\.ts?$': 'ts-jest',
+    "^.+\\.(js|jsx)$": "babel-jest"
```

Comment:
question (blocking): why is this needed? I'd assume that ts-jest could handle all js files and we have no jsx files, only tsx?

Reply:
ah this is left over from something old I was trying. My bad.